t/apicast-bin.t

```diff
@@ -0,0 +1,21 @@
+
+use lib 't';
+use Test::APIcast::Blackbox 'no_plan';
+
+# Can't run twice because one of the test checks the contents of the cache, and
+# those change between runs (cache miss in first run, cache hit in second).
+
+run_tests();
+
+__DATA__
+
+=== TEST 1: Empty APICAST_LOG_LEVEL is ignored.
```

Comment:
Should we say that the default applies rather than it is ignored?

Reply:
Then we should test the default and that's harder than just verifying it works.

Reply:
I'll check it to "does not crash".

Reply:
:+1: